vagrant/base.sh

```diff
@@ -38,8 +38,25 @@ if [ -z `which javac` ]; then
 fi
 
 chmod a+rw /opt
-if [ ! -e /opt/kafka ]; then
-    ln -s /vagrant /opt/kafka
+if [ -h /opt/kafka ]; then
+    # reset symlink
+    rm /opt/kafka
+fi
+ln -s /vagrant /opt/kafka
+
+version=0.8.2.1
+kafka_dir=/opt/kafka-$version
+url=http://psg.mtu.edu/pub/apache//kafka/$version/kafka_2.10-$version.tgz
```

Comment:
Should we host these in our own S3 bucket? I'm sure the apache mirrors are fairly reliable, but it might be better to have something permanent. Alternatively, is there a way to use Apache's mirror lookup script in an automated way, so it just points us to the file instead of giving a page with the list of mirrors?

In fact, now that I've tried to run this, I'm getting an HTML error page for this URL.